src/frontend/verify-jira-jwt-middleware.ts

```diff
@@ -7,6 +7,11 @@ const verifyJiraJwtMiddleware = (tokenType: TokenType) => async (
 	res: Response,
 	next: NextFunction
 ): Promise<void> => {
+	if (!req.query?.xdm_e && !req.body?.jiraHost) {
+		res.status(401);
+		return;
+	}
```

Comment:
We could simply do:
```suggestion
 if(!jiraHost) {
    sendError(res, 401, "Unauthorised");
}
```

after line 15. So we don't have to make changes twice if we have to add another way of getting Jira Host.

(Also everywhere else in this file `sendError` function is used to send error response)